htdocs/admin/system/modules.php

```diff
@@ -277,7 +277,7 @@
 }
 
 // Fields from hook
-$parameters = array('arrayfields'=>$arrayfields, 'param'=>$param, 'sortfield'=>$sortfield, 'sortorder'=>$sortorder);
+$parameters = array('arrayfields'=>$arrayfields, 'param'=>empty($param) ? '' : $param, 'sortfield'=>$sortfield, 'sortorder'=>$sortorder);
```

Comment:
Using test on empty is not always the best way to fix this. Here warning appears because we use a var that is not initialized. A best fix is to add initialization, so add a
`$param = '';`

Another good solution here is to simply remove this param. Because we provide a parameter that does not exists and is never defined into page, so we have no reason to propagate it into the $parameters array.

So here i would simply do:
`$parameters = array('arrayfields'=>$arrayfields, 'sortfield'=>$sortfield, 'sortorder'=>$sortorder);`

I prefer first solution (intialize $param) because it matches the template (the modulebuilder skeleton example) and in future we may add value inside $param